custom_dc/data_cloud.md

```diff
@@ -57,22 +58,22 @@ While you are testing, you can start with a single Google Cloud region; to be cl
 1. For the **Location type**, choose the same regional options as for Cloud SQL above.
 1. When you have finished setting all the configuration options, click **Create**.
 1. In the **Bucket Details** page, click **Create Folder** to create a new folder to hold your data.
-1. Name the folder as desired. Record the folder path as <code>gs://<var>BUCKET_NAME</var>/<var>FOLDER_PATH</var></code> for setting environment variables below. You can start with the sample data provided under `custom_dc/sample` and update to your own data later.
+1. Name the folder as desired. Record the folder path as <code>gs://<var>BUCKET_NAME</var>/<var>FOLDER_PATH</var></code> for setting the `OUTPUT_DIR` environment variable below. 
```

Comment:
Should we mention the deprecated `GCS_DATA_PATH` variable for the benefit of partners using it right now?

Reply:
No, it's in mentioned in the file.